Code/src/App.tsx

```diff
@@ -11,10 +10,13 @@ import {
   setupIonicReact
 } from '@ionic/react';
 import { IonReactRouter } from '@ionic/react-router';
-import { ellipse, square, fastFoodOutline } from 'ionicons/icons';
+import { ellipse, square, triangle, restaurant, person } from 'ionicons/icons';
 import Tab1 from './pages/Tab1';
 import Tab2 from './pages/Tab2';
 import Tab3 from './pages/Tab3';
+import Tab4 from './pages/Tab4';
```

Comment:
We might want to create a "restaurants" folder (./pages/restaurants) and have all the different restaurant tabs in it since it's going to be confusing to have them all in the "pages" folder once we have a lot of restaurants.